src/main/resources/i18n/messages.properties

```diff
@@ -247,9 +247,28 @@ ProjectsTable_create_new_project=Create New Project
 
 
 # Create Project Page
-projects.create.heading=Create a New Project
+server.CreateProject.error=There was an error creating the project.
+CreateProject.title=Create a New Project
```

Comment:
Change this to just `Create New Project`

Reply:
Updated in d2bea5a7f0ff832b38786c6808ca391b5908fa70